lib/rubocop/cop/rails/find_by.rb

```diff
@@ -46,6 +45,12 @@ def on_send(node)
 
         private
 
+        def active_record?(receiver)
```

Comment:
Can you update to the below?

```suggestion
        def where_method?(receiver)
```